stable/spark-operator/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
Chart.lock is the equivalent of requirement.lock in helm3, we only need one of them.

Reply:
Discussed offline, since the chart api version is v2 we must use Chart.lock